lib/AST/DeclContext.cpp

```diff
@@ -300,6 +300,12 @@ PackageUnit *DeclContext::getPackageContext(bool lookupIfNotCurrent) const {
   return nullptr;
 }
 
+bool DeclContext::allowBypassResilienceInPackage() const {
```

Comment:
Is this ever called?

Reply:
yes updated to consolidate into one util func